### PR TITLE
Quantum keycards change color depending on the linked pad's location

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
aka: quantum pads with gags

Quantum keycards now light up depending on the linked pads location/status, and the pads location will be visible on examine.
The card will be white if no pad is linked, the color of the department the linked pad is in, or dark gray as a fallback.
## Why It's Good For The Game
Quantum keycards are impossible to organize, especially if you have multiple on you. Differentiating your secret way into CE's office or space could be vital. This helps to differentiate them and keep you 'safe'.
## Changelog
:cl:
qol: quantum keycards light up with the department theyre in (or grey as a fallback)
qol: quantum keycards are custom renamable w/ pen, to help keep em organized
/:cl:

### CODE REVIEW

**Comment by etherware-novice** [comment]
carg
cc
command
eng
maint
med
not_linked
sci
secoff
srv
heres each variation

File: code/game/objects/items/devices/quantum_keycard.dm [hunk 1]
```diff
@@ -23,8 +46,15 @@
...
+
+	if(!istype(new_pad))
+		set_greyscale(initial(greyscale_colors))
+		name = initial(name)
+		return
 
-/obj/item/quantum_keycard/update_icon_state()
-	icon_state = qpad ? "quantum_keycard_on" : initial(icon_state)
-	return ..()
+	var/new_color = is_type_in_list(get_area(new_pad), gags_coloring, zebra = TRUE) || COLOR_WEBSAFE_DARK_GRAY
```
Comment (Time-Green) [inline code/game/objects/items/devices/quantum_keycard.dm:59]: Why in the god name of fuck does this work

Comment (Time-Green) [inline code/game/objects/items/devices/quantum_keycard.dm:59]: I can't believe this actually fucking works but it actually fucking works

**Review by Time-Green — APPROVED** [review]

File: code/game/objects/items/devices/quantum_keycard.dm (hunk 1 above)
Comment (etherware-novice) [reply code/game/objects/items/devices/quantum_keycard.dm:59]: the magic of `zebra`

**Review by Twaticus — APPROVED** [review]

**Pull request merged**